api/catalog/settings.py

```diff
@@ -298,6 +298,7 @@
 EMAIL_HOST_PASSWORD = config("EMAIL_HOST_PASSWORD", default="")
 EMAIL_SUBJECT_PREFIX = "[noreply]"
 EMAIL_USE_TLS = True
+DEFAULT_FROM_EMAIL = config("DEFAULT_FROM_EMAIL", default="")
```

Comment:
The [Django default](https://docs.djangoproject.com/en/4.0/ref/settings/#default-from-email) is `webmaster@localhost`, did we want to mirror that here so it's more obvious when this isn't set?

Reply:
Personally I'd vote for this group of _required_ email settings to not have defaults unless some parent setting was turned off.

```python
if config("ENABLE_EMAIL", default=True, cast=bool):
    DEFAULT_FROM_EMAIL = config("DEFAULT_FROM_EMAIL")  # no default! This should be an error if you don't supply it
    # etc...
```

Reply:
I love this!